README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
```

Comment:
- [ ]  Your backend project does not have a readme file or a license.  Kindly go [here](https://github.com/microverseinc/readme-template/blob/master/README.md) and follow this template. 👍